client/datasources.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
client needs the memory datastore for maintaining cache of access tokens etc. see comment below.